addons/crm/views/crm_lead_views.xml

```diff
@@ -102,6 +102,12 @@
                                         <field name="is_automated_probability" invisible="1"/>
                                         <field name="probability" widget="float" class="oe_inline"/>
                                         <span class="oe_grey"> %%</span>
+                                        <button
```

Comment:
line 97 : `fa fa-gear` does not exist anymore since font awesome 5.
can you change this in `fas fa-cog` ? in a separated commit ?

Reply:
well well.. they will revert the commits on font awesome 5.0.. so if you already did it, you can drop this change.. :D